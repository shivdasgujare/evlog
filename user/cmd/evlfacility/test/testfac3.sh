#!/bin/sh

# This test test the restricted filter
# Only the message with severity=INFO is logged

TESTNAME=testfac3.sh

/sbin/evlfacility -a "my testfac200" -f 'uid="root" && severity=INFO'
sleep 6
#
#
logtime=`date +%s`
/sbin/evlsend -f "my testfac200" -t 101 -s INFO -m "testfac3:ROOT guy logs this info message"
/sbin/evlsend -f "my testfac200" -t 101 -s WARNING -m "testfac3:ROOT guy logs this warning message"
sleep 1
/sbin/evlview -f "(time>=${logtime} && event_type=101)" -B -S 'size=%size%\nseverity=%severity%\n%data%' > $TESTNAME.out

diff $TESTNAME.out testfac3.tpl.out

if [ $? -eq "0" ]
then 
	echo "testfac3	:PASSED"
	/sbin/evlfacility -d "my testfac200"
else
	echo "testfac3	:FAILED"
fi

