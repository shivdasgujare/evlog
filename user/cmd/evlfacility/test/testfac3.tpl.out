size=41
severity=INFO
testfac3:ROOT guy logs this info message
